pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -903,6 +903,14 @@ func TestTenantStreamingShowTenant(t *testing.T) {
 	defer cleanup()
 	producerJobID, ingestionJobID := c.StartStreamReplication(ctx)
 
+	rowStr := c.DestSysSQL.QueryStr(t, fmt.Sprintf("SHOW TENANT %s WITH REPLICATION STATUS", args.DestTenantName))
```

Comment:
Do you want to port this test over to the datadriven framework now? I also prefer if we had a deterministic test of the different states. I think we can set `stream_replication.job_checkpoint_frequency` to 0 to disable all progress checkpoints. Then we can check that its in initReplication. We can then set the frequency to non-zero and ensure its in replicating and so on?

Reply:
> Do you want to port this test over to the datadriven framework now?

that was the plan but later I thought I want to keep this test.

> I think we can set stream_replication.job_checkpoint_frequency to 0

I'll looks at that, I think this is what I was looking for, thanks!

Reply:
> I think we can set stream_replication.job_checkpoint_frequency to 0

unfortunately this would not help here, because we will still populate the lag info, even when setting that to 0.
let me think about this some more, and we can also discuss offline.